bin/pycbc_live

```diff
@@ -702,34 +702,37 @@ parser.add_argument('--enable-profiling', type=int, metavar='RANK',
                          ' with given rank at the end of program execution')
 
 parser.add_argument('--enable-background-estimation', default=False, action='store_true')
-parser.add_argument('--ifar-upload-threshold', type=float, required=True,
-                    help='Inverse-FAR threshold for uploading coincident '
-                         'triggers to GraceDB, in years.')
 parser.add_argument('--ifar-double-followup-threshold', type=float, required=True,
                     help='Inverse-FAR threshold to followup double coincs with'
                          'additional detectors')
+parser.add_argument('--pvalue-combination-livetime', type=float, required=True,
+                    help="Livetime used for p-value combination with followup "
+                         "detectors, in years")
+parser.add_argument('--enable-single-detector-background', action='store_true', default=False)
+
 parser.add_argument('--enable-gracedb-upload', action='store_true', default=False,
                     help='Upload triggers to GraceDB')
-parser.add_argument('--file-prefix', default='Live')
 parser.add_argument('--enable-production-gracedb-upload', action='store_true', default=False,
                     help='Do not mark triggers uploaded to GraceDB as test '
                          'events. This option should *only* be enabled in '
                          'production analyses!')
-parser.add_argument('--enable-single-detector-upload', action='store_true', default=False)
-parser.add_argument('--pvalue-combination-livetime', type=float, required=True,
-                    help="Livetime used for p-value combination with followup "
-                         "detectors, in years")
-parser.add_argument('--enable-single-detector-background', action='store_true', default=False)
-parser.add_argument('--round-start-time', type=int, metavar='X',
-                    help="Round up the start time to the nearest multiple of X"
-                         " seconds. This is useful for forcing agreement "
-                         " with frame file production.")
+parser.add_argument('--enable-single-detector-upload', action='store_true', default=False,
+                    help='Upload single ifo events to GraceDB')
 parser.add_argument('--gracedb-server', metavar='URL',
                     help='URL of GraceDB server API for uploading events. '
                          'If not provided, the default URL is used.')
 parser.add_argument('--gracedb-search', type=str, default='AllSky',
                     help='String going into the "search" field of the GraceDB '
                          'events')
+parser.add_argument('--ifar-upload-threshold', type=float, required=True,
```

Comment:
The changes in the command line options look to just be a re-ordering. It would be good to revert them to keep the git history looking sensible

Reply:
Actually, the changes are good as they are rationalising the options as in the PR title - ignore previous comment

Reply:
Yes, the intention is to group the FAR calculation vs upload-related options separately.